partner_tier_validation/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Partner Tier Validation",
     "summary": "Support a tier validation process for Contacts",
-    "version": "14.0.2.0.0",
+    "version": "14.0.2.1.0",
```

Comment:
You don't need to do that but you have migration scripts for instance.

It will be done through ocabot

Reply:
Fixed, thank you